### DISCUSSION TITLE
Resources about Qucs-S. Please read this before creating issues.

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#788] [Announcements]
The new documentation for Qucs-S still be under construction #554, but the following help resources are available:

* Documentation web page: https://qucs-s-help.readthedocs.io/en/latest/
* S-parameter viewer and RF circuit synthesis tool documentation (under development): https://andresmmera.github.io/qucs-s-spar-viewer/
* Russian translation of the documentation: [QucsS_RusDoc.pdf](https://github.com/user-attachments/files/24494387/QucsS_RusDoc.pdf)
* Examples pack for documentation: [Shem_for_Documentation.zip](https://github.com/user-attachments/files/24494413/Shem_for_Documentation.zip)
* Qucs-S simulation tutorial https://ra3xdh.github.io/pdf/qucs_s_tutorial.pdf
* Video tutorial from Kasper Nielsen: https://www.youtube.com/watch?v=90RaVy38DB8
* Advanced simulation guide, Part 1: https://habr.com/ru/articles/789452/ Sorry, only Russian version is available.
* Advanced simulation guide, Part 2: https://habr.com/ru/articles/845754/ 
* RF simulation guide for Qucs-S and Ngspice: https://habr.com/ru/articles/735508/ (in Russian only)
* External SPICE models guide: https://habr.com/ru/articles/805963/ (in Russian only) Need an update after #679 
* Other my articles from habr.com (in Russian only): https://habr.com/ru/users/vv_kuznetsov/publications/articles/
* Different example projects using Qucs-S, Ngspice, and QucsatorRF: https://github.com/ra3xdh/qucs_s/discussions?discussions_q=is%3Aopen+label%3Aexamples
* S-parameter viewer and RF circuit synthesis tool documentation: https://andresmmera.github.io/RF-Designer/index.html
* Video tutorial from Stefan Helmert: https://www.youtube.com/watch?v=OzC2PusbDk8&pp=ygUGcXVjcy1z

**Comment by iwbnwif** [comment]
Please can you pin this discussion? It would be good to have it at the top of the list.

**Comment by ra3xdh** [comment]
It is already pinned.

![image](https://github.com/user-attachments/assets/c4449c90-a61c-43de-baf7-28988650afc7)

**Comment by tomhajjar** [comment]
I added a lot of project examples in the Discussions section. Would they be better seen in a new section for examples like ngspice has?

https://sourceforge.net/p/ngspice/discussion/ngspice-tips/

**Comment by ra3xdh** [comment]
Yes, I will categorize the recent examples and add a link to the list.

**Comment by andresmmera** [comment]
I've noticed that the examples provided in the "qucs_s/examples" directory ([1]) are somewhat scattered.

I suggest organizing each example into its own "_prj" folder. This would allow for the inclusion of additional circuit-related materials (for example, a PDF with clarifications and results, which would be quite convenient) alongside each project. If the user copies the examples into their user folder (~/.qucs), this structure would also make it possible to navigate through the projects using the Qucs-S Project View panel.

Furthermore, it might be helpful to classify the examples by category. For instance, files currently located in "qucs_s/examples/qucsator" could be organized as follows:

- RF
  - Amplifiers
  - Filters
    - Active filters
    - Passive filters
      - LPF
      - BPF
      - HPF
      - BSF
  - Oscillators
  - Mixers
  - Couplers
  - Power dividers
  - Noise
  - Miscellaneous
- Power electronics
  - Converters
  - Rectifiers
- General electronics
  - Devices (e.g. transistors, transformers and baluns, gyrator files)
  - Miscellaneous

I can help with this if you're ok with it.

[1] https://github.com/ra3xdh/qucs_s/tree/current/examples

**Comment by ra3xdh** [comment]
Projects are not necessary for Qucs-S operation. I don't use the projects myself except for creating libraries or hierarchical schematics. Furthermore I don't mention projects in the tutorial and don't use project in the practical training of my students at the university.  The sorting example schematic files in the subdirectories would be sufficient.
